geode-core/src/main/java/org/apache/geode/internal/cache/TombstoneService.java

```diff
@@ -733,11 +786,11 @@ public long getScheduledTombstoneCount() {
     /**
      * the expiration time for tombstones in this sweeper
      */
-    protected final long EXPIRY_TIME;
+    protected long EXPIRY_TIME;
```

Comment:
If possible redesign so these could be "final" again. This would require you to know the tombstone timeout when you create a per region sweeper.

If this is not possible then rename these to not be all caps which is reserved for final constants. Since they were being set in the constructor they are not "constants" anyway (even though they were final) so I recommend you change them to be "expiryTime" and "purgeInterval" anyway.